### DISCUSSION TITLE
Dynamically load and render search results when user is scrolling

### DISCUSSION THREAD

**Discussion by PerOve** [microsoft-search/pnp-modern-search#3022] [Ideas]
**Is your feature request related to a problem? Please describe.**
When showing search results that has a lot of results, it is good to use paging.
Paging controls are not always obvious are nice for an end user. Especially on a mobile device.

**Describe the solution you'd like**
It would be nice to just load the batch of hits (next page) dynamically when scrolling, instead of using the paging controls.
It could be controlled just like paging, but with a choice of using dynamic loading based on scrolling.

For Example: 
I have a page with 100 hits in a search. 
On a phone, I would load and render 3-4 articles, and start loading the data for the next 3-4 articles in the background.
When I scroll down the next 4 articles renders and the next 4 articles are loading in the background.

This will be good on low bandwidth devices.
The user experience would be better in my opinion (if anyone care. :))

**Comment by FranckyC** [comment]
Thanks for the suggestion @PerOve. It would require several changes to the code but not impossible. Added it as enhancement.
